test/index.ts

```diff
@@ -0,0 +1,27 @@
+import 'mocha';
+import { startServer } from '../server';
+import axios from 'axios';
+import { expect } from 'chai';
+
+describe('Array', function () {
+  before('starting server', function (done) {
+    startServer().then(done);
+  });
+  it('should get a hello back from the server', async function () {
```

Comment:
uma quebra de linha entre o before e o it poderia dar uma facilitada na leitura hehe